aviary/variable_info/enums.py

```diff
@@ -57,10 +57,9 @@ class EquationsOfMotion(Enum):
     """
     Available equations of motion for use during mission analysis
     """
-    HEIGHT_ENERGY = 'height_energy'
     TWO_DEGREES_OF_FREEDOM = '2DOF'
     # TODO these are a little out of place atm
-    SIMPLE = 'simple'
+    HEIGHT_ENERGY = 'height_energy'
```

Comment:
You made them non-alphabetical now! Plus the TODO is implied to apply to height_energy now, which isn't accurate

Reply:
order moved and comment removed